proto/teleport/lib/teleterm/v1/service.proto

```diff
@@ -61,8 +61,6 @@ service TerminalService {
   // StartHeadlessWatcher starts a headless watcher.
   // If the watcher is already running, it is restarted.
   rpc StartHeadlessWatcher(StartHeadlessWatcherRequest) returns (StartHeadlessWatcherResponse);
-  // GetDatabases returns a filtered and paginated list of databases
-  rpc GetDatabases(GetDatabasesRequest) returns (GetDatabasesResponse);
```

Comment:
I'm not sure what's the best practice when it comes to removing RPCs, it's probably worth asking on Slack. I suppose in a "normal" service we'd deprecate it first, but in teleterm we don't really have to wait for anyone. And the linter doesn't complain about removal so… 🤷.

Reply:
I asked [about it](https://gravitational.slack.com/archives/C0DF0TPMY/p1730987567003429). It's fine here since we ship both the server and the client.